addons/website_sale/static/src/js/website_sale.js

```diff
@@ -19,17 +19,32 @@ $('.oe_website_sale').each(function () {
         });
     });
 
+    $(oe_website_sale).on("change", 'input[name="add_qty"]', function (event) {
+        var qty = $(event.target).closest('form').find('input[name="add_qty"]').val();
+        var product_id = $(event.target).closest('form').find('input[class="product_id"]').val();
+        openerp.jsonRpc("/shop/get_unit_price", 'call', {'product_id': parseInt(product_id),'add_qty': parseInt(qty)})
+        .then(function (data) {
+            $(event.target).closest('form').find(".oe_price .oe_currency_value").text(data.toFixed(2));
+        });
+    });
+
     // change for css
     $(oe_website_sale).on('mouseup touchend', '.js_publish', function (ev) {
         $(ev.currentTarget).parents(".thumbnail").toggleClass("disabled");
     });
 
-    $(oe_website_sale).on("change", ".oe_cart input.js_quantity", function () {
+    $(oe_website_sale).on("change", ".oe_cart input.js_quantity", function (event) {
         var $input = $(this);
         var value = parseInt($input.val(), 10);
         var line_id = parseInt($input.data('line-id'),10);
         if (isNaN(value)) value = 0;
-        openerp.jsonRpc("/shop/cart/update_json", 'call', {
+        openerp.jsonRpc("/shop/get_unit_price", 'call', {
+            'product_id': parseInt($input.data('product-id'),10),
+            'add_qty': value})
+        .then(function (res) {
+
```

Comment:
change price for all product ids (all variants or options) with same quantity and after trigger("change") on ul[data-attribute_value_ids]